### PR TITLE
Allow multi projects

### PR DESCRIPTION
**Pull request by rizqirizqi**
Hi!

I have multiple projects in multiple workspaces. By using your script to clean up one workspace, it will clean up the other workspaces too. I've adjusted your `gemsetcleaner` to be able to check multiple projects in multiple workspaces. So I think I can update yours too. Here's the PR.

### Example
**Usage:**
```
./gemsetcleaner ~/workspace1 ~/workspace2 ~/workspace3
```
**Output:**
```
Detected project gemsets:
/home/mgsrizqi/.rvm/gems/ruby-2.5.0@project3inworkspace1/
/home/mgsrizqi/.rvm/gems/ruby-2.4.2@project2inworkspace2/
/home/mgsrizqi/.rvm/gems/ruby-2.4.1@project7inworkspace2/
/home/mgsrizqi/.rvm/gems/ruby-2.3.0@project4inworkspace3/

A total of 3 unused gemsets were found:

  01) /home/mgsrizqi/.rvm/gems/ruby-2.5.0@project3inworkspace1/
  02) /home/mgsrizqi/.rvm/gems/ruby-2.4.2@project2inworkspace2/
  03) /home/mgsrizqi/.rvm/gems/ruby-2.3.0@project4inworkspace3/

Are you sure you want to remove these 3 gemsets? (yes/no)
```

### CODE REVIEW

File: gemsetcleaner [hunk 1]
```diff
@@ -66,36 +66,44 @@ fi
...
+        
+        # Skip this project if it doesn't contain the .ruby-version and .ruby-gemset
+        # files or a Gemfile
+        if ! [ -f "$project_dir/Gemfile" ] ||
+           ! [ -f "$project_dir/.ruby-version" ] ||
+           ! [ -f "$project_dir/.ruby-gemset" ]; then
+            continue
+        fi
+        
+        ruby_version=$(cat "$project_dir/.ruby-version" | grep -oP '\d\.\d\.\d')
```
Comment (roooodcastro) [inline gemsetcleaner]: I think this REGEX limits the versions too much, for example, Ruby `2.2.10` would not satisfy the REGEX and will get excluded. The same goes for any `rc`/`beta`/`preview` versions, and most importantly, sub versions, like the recent Rails `5.2.4.1` and `6.0.2.1` versions.

Could you expand the REGEXes to accommodate such versions?

**Review by roooodcastro — COMMENTED** [review]
Hi! Thank you for the PR, thanks for adding support for reading the Ruby version from the Gemfile too. Looks good, pending my comment on the REGEX.

File: gemsetcleaner (hunk 1 above)
Comment (rizqirizqi) [reply gemsetcleaner]: nice catch! will update :+1:

Comment (rizqirizqi) [reply gemsetcleaner]: done~~

**Comment by rizqirizqi** [comment]
Hi, it's been 2 years, is there any chance that this MR could be merged? 😅

**Comment by roooodcastro** [comment]
My bad, it's merged now

**Pull request merged**